frontend/public/components/factory/list-page.jsx

```diff
@@ -79,6 +80,7 @@ export class ListPageWrapper_ extends React.PureComponent {
         textFilter={textFilter}
         hideNameFilter={hideNameFilter}
         hideLabelFilter={hideLabelFilter}
+        hideRowFilter={hideRowFilter}
```

Comment:
However, this requires modification in several places as list pages override `rowFilters` prop. 
We could also pass in `filters="HIDE_FILTER"` so that the row filters get overridden internally.

Reply:
I reopened bz1835348 and we will fix that in a different PR.

Reply:
I have raised a PR for 1835348  https://github.com/openshift/console/pull/5481. However, I think we should consider using `filters` props rather than introducing a new prop. :)